share/spice/dota2/dota2.js

```diff
@@ -154,15 +154,34 @@
         var returnData;
         var item = getItemFromData(api_result.itemdata, searchTarget);
         var sourceUrl;
+        var template;
         if (typeof item !== 'undefined') {
             returnData = createItemReturnObject(item, api_result);
             sourceUrl = 'https://www.dota2.com/items/';
+            template = {
+                group: 'base',
+                options: {
+                    content: Spice.dota2.dota2_item
+                }
+            };
         } else {
             var hero = getHeroFromData(api_result.herodata, searchTarget);
             if (typeof hero !== 'undefined') {
                 returnData = createHeroReturnObject(hero, api_result);
             }
             sourceUrl = 'https://www.dota2.com/hero/' + hero.dname.replace(" ", "_");
+            template = {
+                group: 'base',
+                detail: false,
+                'item-detail': false,
```

Comment:
this should be `item_detaill: false`

e.g. https://github.com/duckduckgo/zeroclickinfo-spice/search?utf8=%E2%9C%93&q=item_detail